src/frontend/src/components/AlertAccordion/style.ts

```diff
@@ -1,34 +1,19 @@
 import styled from "@emotion/styled";
 import { Collapse } from "@material-ui/core";
-import { Callout, CommonThemeProps, getIconSizes, getSpaces } from "czifui";
-import ArrowDownIcon from "src/common/icons/IconArrowDownSmall.svg";
-import ArrowUpIcon from "src/common/icons/IconArrowUpSmall.svg";
+import { Callout, getSpaces } from "czifui";
 import { transparentScrollbars } from "src/common/styles/basicStyle";
 
-const smallIcon = (props: CommonThemeProps) => {
-  const iconSizes = getIconSizes(props);
-  return `
-    flex: 0 0 auto;
-    height: ${iconSizes?.s.height}px;
-    width: ${iconSizes?.s.width}px;
-  `;
-};
-
-export const StyledArrowDownIcon = styled(ArrowDownIcon)`
-  ${smallIcon}
-`;
-
-export const StyledArrowUpIcon = styled(ArrowUpIcon)`
-  ${smallIcon}
-`;
-
 export const RowFlexContainer = styled.div`
   display: flex;
   height: 100%;
 
   .MuiCollapse-root {
     overflow-y: auto;
   }
+
+  svg {
+    fill: black;
+  }
```

Comment:
I'm not sure why putting the fill into the `parent component > svg` works while putting the fill elsewhere doesn't seem to work.  I also tried putting it into `smallIcon` above.

Is this an acceptable way to do this?  It seems weird, but I'm not seeing another way.

Reply:
This is the only way to do this atm. There is a small bug in sds.